app/components/views/GovernancePage/Blockchain/AgendaOverview/AgendaOverview.module.css

```diff
@@ -0,0 +1,39 @@
+/* card */
```

Comment:
should this move to a separate css module ?

Reply:
I don't think it is necessary since all classes are related to the card in this file. (This file was not separated originally during dev and I left this comment by accident.)